api/v1alpha1/monitoringconfig_types.go

```diff
@@ -0,0 +1,79 @@
+// everest-operator
+// Copyright (C) 2022 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// MonitoringType is a type of monitoring.
+type MonitoringType string
+
+const (
+	// PMM represents monitoring via PMM.
+	PMM MonitoringType = "pmm"
```

Comment:
I usually prefer `PMM` instead of `PMMMonitoringType` as I feel it's too verbose and `PMM` has a type of `MonitoringType`.
But I'm happy to change it. It's only a matter of personal preference.